var/spack/repos/builtin/packages/py-timm/package.py

```diff
@@ -32,15 +33,15 @@ class PyTimm(PythonPackage):
     with default_args(type=("build", "run")):
         # https://github.com/huggingface/pytorch-image-models/issues/1530
         # https://github.com/huggingface/pytorch-image-models/pull/1649
-        depends_on("python@:3.10", when="@:0.6.12", type=("build", "run"))
+        depends_on("python@:3.10", when="@:0.6.12")
 
-        depends_on("py-torch@1.7:", when="@0.6:", type=("build", "run"))
-        depends_on("py-torch@1.4:", type=("build", "run"))
-        depends_on("py-torchvision", type=("build", "run"))
-        depends_on("py-pyyaml", when="@0.6:", type=("build", "run"))
-        depends_on("py-huggingface-hub", when="@0.6:", type=("build", "run"))
-        depends_on("py-safetensors", when="@0.9:", type=("build", "run"))
+        depends_on("py-torch@1.7:", when="@0.6:")
+        depends_on("py-torch@1.4:")
+        depends_on("py-torchvision")
+        depends_on("py-pyyaml", when="@0.6:")
+        depends_on("py-huggingface-hub", when="@0.6:")
+        depends_on("py-safetensors", when="@0.9:")
 
         # https://github.com/rwightman/pytorch-image-models/pull/1256
-        depends_on("pil@:9", when="@:0.5", type=("build", "run"))
-        depends_on("py-numpy", when="@:0.5", type=("build", "run"))
+        depends_on("pil@:9", when="@:0.5")
+        depends_on("py-numpy", when="@:0.5")
```

Comment:
Why are the explicit dependency types being removed here?

Reply:
Because this whole section is indented within a:
```python
with default_args(type=("build", "run")):
```
making these redundant.

Reply:
🤦 How did I miss that?!!!!

Reply:
I saw comments and completely overlooked line 33.

Reply:
Lesson: Be **extra** careful when you aren't feeling well.